lib/common/common/src/validation.rs

```diff
@@ -3,6 +3,9 @@ use std::borrow::Cow;
 use serde::Serialize;
 use validator::{Validate, ValidationError, ValidationErrors};
 
+// Multivector sholud be small enough to fir the chunk of vector storage
+pub const MAX_MULTIVECTOR_FLATTENED_LEN: usize = 1_000_000;
```

Comment:
```suggestion
pub const MAX_MULTIVECTOR_FLATTENED_LEN: usize = 1024 * 1024;
```

Reply:
Also, `_LEN` means it will be the same for bytes / half-floats / floats?

Reply:
Also, this should be documented

Reply:
We validate at the point where we don't know vector storage type. So I have chosen the worse option - `f32` and set 1M elements

Reply:
Why not set the value to be in bytes?